openmicroscopy/openmicroscopy
# Big image export jpg 12982 develop

This is #4172 re-based to develop:

---------------------------------

This limits the download as jpeg / png / tiff to images of 12k * 12k or smaller.
This cut-off was determined by trying to download images of various sizes.

To test:
 - Images bigger than 12k * 12k pixels (144000000) should be disabled for Download as Jpg, Png & Tiff.
 - Configure with a different number of pixels. E.g. 3000 x 3000 = 9000000
```
$ bin/omero config set omero.client.download_as.max_size 9000000
$ bin/omero admin restart
```
Now, images larger than this new size should be disabled for download.

NB: The 5.1 PR #4172 doesn't include the last 'revert' commit that adds the config parameter above. Could rebase this if we want to?

--rebased-from #4172
